pom.xml

```
@@ -8,8 +8,8 @@
   </parent>
 
   <properties>
-    <revision>3.1</revision>
-    <changelist>-SNAPSHOT</changelist>
+    <revision>3</revision>
```

```suggestion
    <revision>4</revision>
```
If you want to use a manually controlled prefix, this must be set to `4` to prevent a maven mismatch with our custom format.